samples/sensor/htu21d/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2020, Philémon Jaermann
```

Comment:
The sample should be named `samples/sensor/sht21` since that's the name of the driver, even if the in-tree example uses an HTU21D.